Jenkinsfile

```diff
@@ -72,18 +72,20 @@ node('boardfarm') {
         }
 
         def dl_path = "${image_path}/${image_name}"
-        sh "wget '${dl_path}' -O ${env.WEBSERVER_PATH}/openwrt.ubi"
 
         /* Reconfigure the DUT in case it came back from a reboot and no networking
            is available */
         configureDut();
 
+        checkout scm
```

Comment:
Suggest adding a new stage for "Update DUT" (or similar wording) which include the checkout step and running the ota scripts, this will allow us to analyse the time taken for the update process in this stage, and further optimise it